Dockerfile

```diff
@@ -16,6 +16,8 @@ COPY tdm-fe /app/
 # Build the React app
 RUN npm run build
 
+#######################################################
+
 # Stage 2: Serve React App using Express
 FROM node:lts-slim
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏fix node version